absl/base/internal/sysinfo.cc

```diff
@@ -95,6 +101,8 @@ static double GetNominalCPUFrequency() {
     }
   }
   return 1.0;
+#endif /* WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_APP) && \
+          !WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP) */
```

Comment:
LINT warning: 
Complex multi-line /*...*/-style comment found. Lint may give bogus warnings.  Consider replacing these with //-style comments, with #if 0...#endif, or with more clearly structured multi-line comments.

Reply:
I'm not familiar with the linting rules used by abseil-cpp.
What is the recommended way to have that comment?  Would the following acceptable?
```c
#endif // WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_APP) && \
       // !WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP)

Reply:
Fixed: #endif // WINAPI_PARTITION_APP && !WINAPI_PARTITION_DESKTOP